### ISSUE TITLE
Secrets in Backup/Restore should be only opaque

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#2405]
Noted in Rancher 2.5.3, Secrets in Backup/Restore should be only opaque secrets, grouped by namespace, ordered alphabetically (use secretselector component, needs to support multiple namespaces for this case).

Secrets aren’t in any sensible alphabetical order. We could probably do this alphabetical by namespace, but namespaces aren’t shown. Since the key is at the bottom, so maybe by creation date? Or sortable?

**Comment by nickwsuse** [comment]
@eva-vashkevich I have a clarifying question about this. There's an info icon in the `Encryption Config Secret` that seems to suggest that the dropdown will only show secrets in the `cattle-resource-system` namespace (also a small typo as it's `cattle-resources-system`), and only those that have an `encryption-provider-config.yaml` key. I created an opaque secret in the `default` namespace and in `cattle-resources-system` and added the key in the info icon and they both appear.

TL;DR - Should this actually be pulling from all namespaces and the helper text should be updated (or made more clear), or is this only supposed to be pulling from the `cattle-resources-system` namespace?

![image](https://github.com/rancher/dashboard/assets/107147272/d68f17c7-b83c-4b70-ba03-e058cfc4b5bf)

**Comment by eva-vashkevich** [comment]
Hi @nickwsuse 
Based on https://github.com/rancher/dashboard/pull/1179 it should be only pulling from the cattle-resources-system namespace. Thank you for catching this.

**Comment by eva-vashkevich** [comment]
Hi @nickwsuse I don't seem to see any secrets from default namespace locally. Could you please share the steps to reproduce it?
Thank you

**Comment by nickwsuse** [comment]
@eva-vashkevich It should be the first field when creating an opaque secret under local -> storage -> secrets -> create -> opaque
![image](https://github.com/rancher/dashboard/assets/107147272/5bfdadad-f4ab-4b8c-9d46-f9f34cf5fedd)

**Comment by eva-vashkevich** [comment]
![Screenshot 2023-09-25 at 6 02 10 PM](https://github.com/rancher/dashboard/assets/133266076/2a2851eb-f7b1-454b-b593-6bda5abdbb9c)
![Screenshot 2023-09-25 at 6 02 35 PM](https://github.com/rancher/dashboard/assets/133266076/c2efb74f-fe3e-4e7f-8095-8b7da37060a5)

Anything else you might be doing different?

**Comment by nickwsuse** [comment]
@eva-vashkevich Hmm, I'm not seeing secrets from the default namespace now either. I followed the same steps I did previously (left the namespace as default and added encryption-provider-config.yaml as one of the keys) but now I only see secrets from the cattle-resources-system namespace. I'm on v2.8-head and was previously when I was seeing other secrets.

![image](https://github.com/rancher/dashboard/assets/107147272/0db47799-4887-492c-b1e4-9c9f40d299b7)
![image](https://github.com/rancher/dashboard/assets/107147272/a1e1526d-50fb-45be-ba1a-67dbb0543891)

**Comment by eva-vashkevich** [comment]
I've opened a PR to fix the typo in the tooltip, but I am not sure why you were getting secrets from default namespace before. I would suggest we keep an eye on it for now until we see some way to reproduce it.

Thank you

**Comment by nickwsuse** [comment]
`Verified on v2.8-head ID: 4c55a12`

Still not sure what I did differently to get secrets outside of the `cattle-resources-system` namespace to appear the first time, but I'm only seeing opaque secrets with the `encryption-provider-config.yaml` as one of the keys. Non-opaque secrets aren't appearing, nor are opaque secrets _without_ `encryption-provider-config.yaml` as a key.
